server.go

```diff
@@ -131,8 +132,13 @@ func initAggregationRules(specifications []models.AggregationSpecification) []ag
 	var rules = make([]aggregation.Rule, len(specifications))
 	i := 0
 	for _, spec := range specifications {
-		rules[i] = aggregation.NewAggregationRule(spec)
-		i++
+		rule, err := aggregation.NewAggregationRule(spec)
+		if !zero.IsZero(rule) && err == nil {
```

Comment:
If you check for err != nil, then you can simplify this if statement

Reply:
Yes! Thanks! 👍